plover/gui_qt/add_translation_dialog.py

```diff
@@ -11,6 +11,7 @@ class AddTranslationDialog(Tool, Ui_AddTranslationDialog):
     ''' Add a new translation to the dictionary. '''
 
     TITLE = _('Add Translation')
+    TOOLTIP = _('Add a new translation to the dictionary')
```

Comment:
How about this instead:
```python
    __doc__ = _('Add a new translation to the dictionary.')
```
?

Reply:
Docstrings used to be read-only, but are changeable now. But their goal is to work as a type of comments that can be fetched automatically when you're trying to call a function but want to get reminded what the function does or how it's meant to be used. By encoding them in an unusually way, we're stopping IDEs from showing them, but fetching the docstring is just fetching the __doc__ attribute, so creating an attribute called TOOLTIP means anyone looking at the code can guess what that tooltip is rather than wondering why we're setting the docstring that way.